doc/en/components/charts/chart-types-bar.md

```diff
@@ -6,23 +6,118 @@ mentionedTypes: ["XamDataChart"]
 
 ## $PlatformShort$ Stacked Bar Chart
 
-<!-- TODO copy and combine content (code snippets, description) from these topics:
-	data-chart-type-stacked-bar-series.md
-    data-chart-type-stacked-100-bar-series.md
--->
+A Stacked Bar Chart, or Stacked Bar Graph, is a type of category chart that is used to compare the composition of different categories of data by displaying different sized fragments in the horizontal bars of the chart. The length of each bar, or stack of fragments, is proportionate to its overall value. 
+
+The $PlatformShort$ Stacked Bar Chart differs from the $PlatformShort$ Bar Chart in that the data points representing your data are stacked next to each other horizontally to visually group data your data. Each stack can contain both positive and negative values. All positive values are grouped on the positive side of the X-Axis, and all negative values are grouped on the negative side of the X-Axis.
+
+### $PlatformShort$ Stacked Bar Chart Example 
+
+In this example of an $PlatformShort$ Stacked Bar Chart, we have a Numeric X Axis (bottom labels of the chart) and a Category Y Axis (left labels of the chart).
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Stacked 100 Bar Chart
+
+The $PlatformShort$ Stacked 100 Bar Chart is identical to the $PlatformShort$ stacked bar chart in all aspects except in their treatment of the values on X-Axis (bottom labels of the chart). Instead of presenting a direct representation of the data, the stacked 100 bar chart presents the data in terms of percent of the sum of all values in a data point.
+
+### $PlatformShort$ Stacked 100 Bar Chart Example 
+
+In this example of a Stacked 100 Bar Chart, the Energy Product values are shown as a 100% value of all of the data in the fragments of the horizontal bars.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Range Bar Chart
+
+The Ignite UI for $PlatformShort$ range column chart belongs to a group of range charts and is rendered using two lines with the area between the lines filled in. This type of series emphasizes the amount of change between low values and high values in the same data point over a period of time or compares multiple items. Range values are represented on the y-axis and categories are displayed on the X-Axis.
+
+### $PlatformShort$ Range Bar Chart Example 
+
+In this Range Bar Chart example, weather data variance between high and low temperatures by date is shown.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## Styling the $PlatformShort$ Bar Chart
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=1eac53a1-a250-4bfe-947f-047bf7b8a0f4&mode=design

Reply:
Maybe you can also show the chart here in a color different than the series first color, just as an idea :)

Reply:
https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=0a127863-0c63-4a1f-b335-b70e5179f263&mode=design